style.css

```diff
@@ -74,6 +74,151 @@ a {
   padding: 10px 5px;
 }
 
+/**  mobile  popup */
+
+.splash {
+  position: fixed;
+  backdrop-filter: blur(4px);
+  background-color: rgba(193, 199, 208, 0.5);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 24px;
+}
+
+.splash ul.due-time {
+  display: flex;
+  padding-left: 0;
+}
+
+.popup {
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  flex-basis: 100%;
+  display: flex;
+  padding: 2.4em 2.4em 6.6em  2.4em;
+  margin: 2% auto;
+  border-radius: 30px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  background-color: white;
+  height: 94%;
+  overflow-y: auto;
+  flex-direction: column;
+}
+
+.popup .pop-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.popup .work-item-list {
+  background: white;
+  border-radius: 40px;
+  padding: 12px;
+}
+
+.popup .popup-image {
+  max-height: 60%;
+  padding-bottom: 4em;
+}
+
+.popup .info {
+  display: flex;
+  flex-direction: column;
+  row-gap: 14px;
+}
+
+.popup .live-icon {
+  min-width: 0;
+  width: 18px;
+  height: 18px;
+  margin: 0 0 0 12px;
+  padding-top: 5px;
+}
+
+.popup .tech-tags {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 14px;
+  column-gap: 14px;
+  border-bottom: solid 0.1em #ebecf0;
+  margin-left: 12px;
+  padding-left: 0;
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+}
+
+.tech-tags li {
+  background-color: #ebebff;
+  border-radius: 5px;
+  color: #6070ff;
+  padding: 4px 12px;
+  font-size: 12px;
+}
+
+.popup .link-tags {
+  margin-left: 12px;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 14px;
+}
+
+.popup .text {
+  flex-basis: 90%;
+  max-width: 320px;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.popup aside span {
+  padding: 0 12px;
+  display: flex;
+  column-gap: 10px;
+}
+
+.bu-fl {
+  display: flex;
+  border: 1px solid #6070ff;
+  margin-right: 10px;
+  padding: 12px 22px;
+  font-size: 15px;
+  border-radius: 0.5em;
+  align-items: center;
+}
+
```

Comment:
- [ ] Nice work styling the popup's buttons :computer: :sunglasses: however they look different compared to the Figma Design( see images ). Kindly make sure you follow the Figma Design.

### Your Design :point_down: 
![Screenshot from 2023-04-22 18-29-08](https://user-images.githubusercontent.com/91142432/233801134-f7f8e840-3f5f-49a1-80b2-3487b87de096.png)

### Figma Design :point_down: 
![Screenshot from 2023-04-19 13-30-07](https://user-images.githubusercontent.com/91142432/233801157-756be818-9d85-4055-9453-7bf553fa5736.png)